Description: Use DejaVu instead of Bitstream Vera
 ttf-bitstream-vera was removed from the archive in favor of ttf-dejavu. Update
 rxvt-unicode accordingly.
 .
 Not forwarding upstream since this is an esthetic change.
Author: Decklin Foster <decklin@red-bean.com>
Origin: vendor
Bug-Debian: http://bugs.debian.org/461289
Forwarded: no
Reviewed-By: Ryan Kavanagh <ryanakca@kubuntu.org>
Last-Update: 2011-02-24
Index: pkg-urxvt/src/rxvtfont.C
===================================================================
--- pkg-urxvt.orig/src/rxvtfont.C	2011-07-19 14:40:16.136549809 -0400
+++ pkg-urxvt/src/rxvtfont.C	2011-07-19 14:40:19.120564605 -0400
@@ -136,7 +136,7 @@
 
   //{ CS_UNICODE,      "-*-unifont-*-*-*-*-*-*-*-*-c-*-iso10646-1"   }, // this gem of a font has actual dotted circles within the combining character glyphs.
 #if XFT
-  { CS_UNICODE,      "xft:Bitstream Vera Sans Mono:antialias=false:autohint=true" },
+  { CS_UNICODE,      "xft:DejaVu Sans Mono:antialias=false:autohint=true" },
   { CS_UNICODE,      "xft:Courier New:antialias=false:autohint=true"              },
   { CS_UNICODE,      "xft:Andale Mono:antialias=false:autohint=false"             },
   { CS_UNICODE,      "xft:Arial Unicode MS:antialias=false:autohint=false"        },
